tests/fieldmodule/fieldparameterstests.cpp

```diff
@@ -47,6 +47,10 @@ TEST(ZincFieldparameters, validAPI)
 	EXPECT_TRUE(fieldOut.isValid());
 	EXPECT_EQ(coordinates, fieldOut);
 
+	EXPECT_DOUBLE_EQ(0.0, fieldparameters.getTime());
+	EXPECT_EQ(RESULT_OK, fieldparameters.setTime(1.234));
```

Comment:
In the documentation we say:
` * @param timeIn  The time to set. Must match a time held at nodes.`
Is the time `1.234` a time held at the nodes for the cube used here?

Reply:
No it doesn't; I'm just testing the call as simply as I can knowing it isn't currently checked.

Reply:
I think we should change to documentation from ` Must match a time held at nodes.` too `In the future, timeIn must match a time held at nodes.`